src/indicators/spinner.js

```diff
@@ -31,7 +31,7 @@ const propTypes = {}
 const defaultProps = {}
 
 function Spinner (props) {
-  return <div id="spinner" { ...filterInvalidDOMProps(props) } />
+  return <div className="spinner" { ...filterInvalidDOMProps(props) } />
```

Comment:
I think we should allow for the user to also specify an additional classname. We can do this using the `classnames` package. You can reference an example of this in `src/forms/buttons/button-area.js`.